script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -1183,7 +1183,7 @@ Ropes of the woman's red blood stretch between my feet and the floor like coal t
 [sha:72d6f5cd8a93b470c1e7b496ef0c010178212d0c]{
 -- Page 244, Offset 4653.
 -- ああァ、アア、あかい、ち、だ。
-Rrr-re-re-red... red... blood.
+R-R-R-Re-Re-Red... red... blood.
```

Comment:
Is this better @rschlaikjer? Rrr reads like he's growling or something

Reply:
Sure